ui/src/notebooks/components/panel/NotebookPanel.tsx

```diff
@@ -15,6 +15,9 @@ import PanelVisibilityToggle from 'src/notebooks/components/panel/PanelVisibilit
 import MovePanelButton from 'src/notebooks/components/panel/MovePanelButton'
 import NotebookPanelTitle from 'src/notebooks/components/panel/NotebookPanelTitle'
 
+// Constants
+import {NOTEBOOK_PANEL_ID} from 'src/notebooks/constants'
+
```

Comment:
I made this a constant cuz I am using it in 2 places and I didn't want them to drift apart